api/dev/library/OpenRA.lua

```diff
@@ -1234,16 +1239,17 @@ local __actor = {
     ---@param damageTypes? any
     Kill = function(damageTypes) end;
 
-    --- Current health of the actor.
-    --- **Requires Trait:** [IHealth](https://docs.openra.net/en/release/traits/#ihealth)
-    ---@type integer
-    Health = nil;
-
     --- Maximum health of the actor.
     --- **Requires Trait:** [IHealth](https://docs.openra.net/en/release/traits/#ihealth)
     ---@type integer
     MaxHealth = nil;
 
+    --- Enter the target actor to repair it instantly.
+    --- *Queued Activity*
+    --- **Requires Traits:** [IMove](https://docs.openra.net/en/release/traits/#imove), [InstantlyRepairs](https://docs.openra.net/en/release/traits/#instantlyrepairs)
```

Comment:
```suggestion
    --- **Requires Traits:** [Mobile](https://docs.openra.net/en/release/traits/#Mobile), [InstantlyRepairs](https://docs.openra.net/en/release/traits/#instantlyrepairs)
```

or maybe just unlink it.